exonum-java-binding-core/rust/src/runtime/config.rs

```diff
@@ -73,6 +75,7 @@ fn check_not_forbidden(user_parameter: &str) -> Result<(), ForbiddenParameterErr
     if user_parameter.starts_with("Djava.class.path")
         || user_parameter.starts_with("Djava.library.path")
         || user_parameter.starts_with("Dlog4j.configurationFile")
+        || user_parameter.starts_with("agentlib:jdwp")
```

Comment:
There is no reason to forbid it

Reply:
Well, I got it as here we ensure that nothing that is passed via other arguments is passed via `ejb-jvm-args`. So I just tried to follow the initial idea.